lib/diagnostic.js

```diff
@@ -60,7 +61,9 @@ let deathStar = {
 // Question 4: Write the code that would set the `deathStar`'s `fire` method
 // in the context of `alderaan` but does not invoke the method,
 // so that we could call answer4() // Alderaan
-let answer4 = null;  // replace null with your response
+let answer4 = deathStar.fire(); //not sure if alderaan should be a parameter passed to fire
```

Comment:
Similar to the last one, but instead of calling the method, we want to bind it so that when `answer4` is called, it returns `alderaan`